README.md

```diff
@@ -57,6 +57,7 @@ environments.
 | Ceph CSI Version | Container Orchestrator Name | Version Tested|
 | -----------------| --------------------------- | --------------|
 | v3.9.0 | Kubernetes | v1.25, v1.26, v1.27|
+| v3.8.1 | Kubernetes | v1.24, v1.25, v1.26, v1.27|
```

Comment:
check if we are running tests with 1.24 or not.

Reply:
It's triggered but not mandated to have k8s v1.24 by mergifyio  
https://github.com/ceph/ceph-csi/pull/4005 
updates to remove that version,
ptal